changelogs/fragments/5533-add-merge-variables-lookup-plugin.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - merge_variables - Add new merge_variables lookup plugin (https://github.com/ansible-collections/community.general/pull/5533).
```

Comment:
New plugins do not need a changelog fragment. What is needed is an entry in `.github/BOTMETA.yml` though.

Reply:
Changelog removed and `BOTMETA.yml` updated